### PR TITLE
Partthree

### PR DESCRIPTION
**Pull request by harringtonben**

### CODE REVIEW

File: javascripts/controllers/NewCtrl.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+			"last_name": $scope.newcontact.lastname,
+			"phone_number": $scope.newcontact.phonenumber,
+			"email": $scope.newcontact.email,
+			"twitter": $scope.newcontact.twitter,
+			"facebook_page": $scope.newcontact.facebook,
+            "instagram_username": $scope.newcontact.instagram,
+            "user_id": $rootScope.uid
+        };
+
+        DatabaseService.addNewContact(newContact);
```
Comment (morecallan) [inline javascripts/controllers/NewCtrl.js]: Because you aren't using .then() here, the contacts aren't loading but the time it switches location.

**Review by morecallan — COMMENTED** [review]
@harringtonben - mostly there! There is an async issue that I pointed out. Go ahead and get that fixed and pushed up and I'll merge it!

File: javascripts/controllers/NewCtrl.js (hunk 1 above)
Comment (harringtonben) [reply javascripts/controllers/NewCtrl.js]: @morecallan updated!

**Comment by morecallan** [comment]
💯 🍾 🎉

**Pull request merged**